services/web/client/source/class/osparc/data/Permissions.js

```diff
@@ -166,6 +166,7 @@ qx.Class.define("osparc.data.Permissions", {
           "study.nodestree.uuid.read",
           "study.filestree.uuid.read",
           "study.logger.debug.read",
+          "study.slides",
```

Comment:
For the moment this feature is in the front-end. How do you envision syncing permissions between front-end/backend?